virtualenv_embedded/activate.sh

```diff
@@ -53,12 +53,23 @@ if ! [ -z "${PYTHONHOME+_}" ] ; then
     unset PYTHONHOME
 fi
 
+newline_regex='(^
++)(.*)'
+
 if [ -z "${VIRTUAL_ENV_DISABLE_PROMPT-}" ] ; then
     _OLD_VIRTUAL_PS1="$PS1"
     if [ "x__VIRTUAL_PROMPT__" != x ] ; then
         PS1="__VIRTUAL_PROMPT__$PS1"
     else
-        PS1="(`basename \"$VIRTUAL_ENV\"`) $PS1"
+        env_basename=`basename "$VIRTUAL_ENV"`
+        set +u
```

Comment:
Setting this so that `tests/test_activate.sh` won't fail on an unbound variable `ZSH_VERSION` when running on Bash where it is supposed to be unset.